packages/pcd/gpc-pcd/test/gpc-pcd.spec.ts

```diff
@@ -187,4 +167,212 @@ describe("GPCPCD should work", async function () {
   });
 });
 
+describe("GPCPCD input POD validator should work", () => {
```

Comment:
This is good test layering, thanks.
This layer is ensuring that each individual validator is properly wired into the PCD layer, while the tests in validatorChecks.spec.ts are able to give more detailed coverage to each checker.